### ISSUE TITLE
node.napi.musl.node missing a symbol

### ISSUE THREAD

**Issue by TheGeolama** [BenjaminPrice/fvtt-docker#35]
When I attempt to use the latest FoundryVTT 11.301 zip file, I see this in the fvtt container's log:

  inflating: snapshot_blob.bin,
  inflating: v8_context_snapshot.bin,
  inflating: vk_swiftshader_icd.json,
node:internal/modules/cjs/loader:1168,
  return process.dlopen(module, path.toNamespacedPath(filename));,
                 ^,
,
Error: Error relocating /opt/foundryvtt/resources/app/node_modules/classic-level/prebuilds/linux-x64/node.napi.musl.node: _ZSt28__throw_bad_array_new_lengthv: symbol not found,
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1168:18),
    at Module.load (node:internal/modules/cjs/loader:989:32),
    at Function.Module._load (node:internal/modules/cjs/loader:829:14),
    at Module.require (node:internal/modules/cjs/loader:1013:19),
    at require (node:internal/modules/cjs/helpers:93:18),
    at load (/opt/foundryvtt/resources/app/node_modules/node-gyp-build/node-gyp-build.js:22:10),
    at Object.<anonymous> (/opt/foundryvtt/resources/app/node_modules/classic-level/binding.js:1:43),
    at Module._compile (node:internal/modules/cjs/loader:1109:14),
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10),
    at Module.load (node:internal/modules/cjs/loader:989:32) {,
  code: 'ERR_DLOPEN_FAILED',

This might be a problem with the Alpine Linux Node 16 build not using glibc

**Comment by kbfoerster** [comment]
Not sure if you were able to resolve this, but I ran into this same error and was able to resolve it by switching my foundryvtt container to [felddy/foundryvtt-docker](https://github.com/felddy/foundryvtt-docker). 

When I hit this error I was running an older version of this container from late 2021. I tried updating, but was hitting the permissions error in #33 . Rather than resolving those errors, I just swapped over to the other container and was able to be up and running pretty quick with no loss of data. 

My server was originally set up using the [steps on Ben's blog](https://benprice.dev/posts/fvtt-docker-tutorial/). To swap to the new container, I just stopped this one, added the new one, and copied over all the labels and other configs to the new one. Only changes were I just added the one `/data/` volume mount that had all my existing world data and added the authentication env vars to pull down the latest foundryvtt version.

Sorry if this is lame to advocate for another container, but this one doesn't seem actively maintained while the other is. Will happily remove this if asked, but wanted to put this out there if you're blocked from upgrading by this error.
